overlays/DIMO/dimo.vspec

```diff
@@ -71,3 +71,15 @@ Vehicle.IsIgnitionOn:
   type: sensor
   datatype: boolean
   description: Vehicle ignition status. False - off, True - on.
+
+Vehicle.Powertrain.TractionBattery.Charging.ChargeCurrent.AC:
+  datatype: float
+  type: sensor
+  unit: A
+  description: Current AC charging current (rms) at inlet. Negative if returning energy to grid. Used when per-phase numbers are unavailable.
+
+Vehicle.Powertrain.TractionBattery.Charging.ChargeVoltage.UnknownType:
```

Comment:
@KevinJoiner @lernerjn Please help

Reply:
I like unkownType because no one is going to use this without looking at the description.